hw/ip/csrng/doc/checklist.md

```diff
@@ -40,25 +40,25 @@ Code Quality  | [LINT_SETUP][]                 | Done        |
 
 Type          | Item                    | Resolution  | Note/Collaterals
 --------------|-------------------------|-------------|------------------
-Documentation | [NEW_FEATURES][]        | Not Started |
-Documentation | [BLOCK_DIAGRAM][]       | Not Started |
-Documentation | [DOC_INTERFACE][]       | Not Started |
-Documentation | [MISSING_FUNC][]        | Not Started |
-Documentation | [FEATURE_FROZEN][]      | Not Started |
-RTL           | [FEATURE_COMPLETE][]    | Not Started |
-RTL           | [AREA_CHECK][]          | Not Started |
-RTL           | [PORT_FROZEN][]         | Not Started |
-RTL           | [ARCHITECTURE_FROZEN][] | Not Started |
-RTL           | [REVIEW_TODO][]         | Not Started |
-RTL           | [STYLE_X][]             | Not Started |
-Code Quality  | [LINT_PASS][]           | Not Started |
-Code Quality  | [CDC_SETUP][]           | Not Started |
-Code Quality  | [FPGA_TIMING][]         | Not Started |
-Code Quality  | [CDC_SYNCMACRO][]       | Not Started |
+Documentation | [NEW_FEATURES][]        | Done        |
+Documentation | [BLOCK_DIAGRAM][]       | Done        |
+Documentation | [DOC_INTERFACE][]       | Done        |
+Documentation | [MISSING_FUNC][]        | Done        |
+Documentation | [FEATURE_FROZEN][]      | Done        |
+RTL           | [FEATURE_COMPLETE][]    | Done        |
+RTL           | [AREA_CHECK][]          | Done        |
+RTL           | [PORT_FROZEN][]         | Done        |
+RTL           | [ARCHITECTURE_FROZEN][] | Done        |
+RTL           | [REVIEW_TODO][]         | Pending     |
+RTL           | [STYLE_X][]             | Done        |
+Code Quality  | [LINT_PASS][]           | Done        |
+Code Quality  | [CDC_SETUP][]           | N/A         |
```

Comment:
just to clarify, are there no CDCs in these three modules at all?

Reply:
True, except there is the "lc_sync" module in csrng. I was assuming this was self-contained and did not need additional CDC analysis. Is this the general approach taken?

Reply:
Ok good, just wanted to double check.

I believe the CDC_SETUP is actually meant to stand for "tool setup", in the same way as the LINT_SETUP checklist item in D1.

Since we do not have a CDC tool set up yet, it is ok to waive this at this time, but I wonder whether we should mark this either as `waived` or `not started`, since in principle the item applies as there are CDCs within this module.

@eunchan @tjaychen WDYT?

Reply:
Can you change the `CDC_SETUP` items in these checklists to:
```
Code Quality  | [CDC_SETUP][]           | Waived      | CDC flow is not available yet. 
```

Reply:
thansks @msfschaffner - updated only CSRNG, since EDN and ES do not have CDC crossings.